kotlinx-coroutines-core/concurrent/test/flow/StateFlowCommonStressTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.flow
+
+import kotlinx.coroutines.*
+import kotlin.random.*
+import kotlin.test.*
+
+// A simplified version of StateFlowUpdateStressTest
```

Comment:
I think this goes to `StateFlowUpdateCommonTest`, whereas this is a simplified version of `StateFlowStressTest`.